### PR TITLE
Added FAQ section to team page

### PR DESCRIPTION
**Pull request by EinarBalan**
Ripped directly from hoth website ty

lmk if more questions should be added/answers should be tweaked.

### CODE REVIEW

File: src/components/PhotoPage/FAQSection.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+import Typography from '@material-ui/core/Typography';
+import MuiAccordion from '@material-ui/core/Accordion';
+import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
+import MuiAccordionDetails from '@material-ui/core/AccordionDetails';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+import Add from '@material-ui/icons/Add';
+import Remove from '@material-ui/icons/Remove';
+
+// Stolen from HOTH website, yoink
```
Comment (khxia) [inline src/components/PhotoPage/FAQSection.js:17]: let's remove this LMAO

**Review by khxia — CHANGES_REQUESTED** [review]

File: src/components/PhotoPage/FAQSection.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+				<>
+					We recommend attending workshops and events (such as HOTH or Hack Kitchen)
+					, joining a Hack Fam, or asking us questions in the ACM discord!
+				</>
+		},
+		{
+			question: `Your past workshops seem interesting, how can I learn more?`,
+			answer:
+				<>
+					Check out the <Link href="https://hack.uclaacm.com/archive">archive</Link> page!
```
[the remark this answers is not in the document]
Comment (TimothyGu) [reply src/components/PhotoPage/FAQSection.js:135]: Also, please use relative URLs. This way during development we can visit the archive page belonging to the local instance.

```suggestion
					Check out the <Link href="/archive">archive</Link> page!
```

File: src/components/PhotoPage/FAQSection.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+				<>
+					We post all recordings of our workshops on the ACM youtube channel, which can be found <Link href="https://www.youtube.com/c/ACMUCLA/videos">here</Link>!
+				</>
+		},
+		{
+			question: `How do I participate?`,
+			answer:
+				<>
+					We recommend attending workshops and events (such as HOTH or Hack Kitchen)
+					, joining a Hack Fam, or asking us questions in the ACM discord!
```
Comment (TimothyGu) [inline src/components/PhotoPage/FAQSection.js:128]:
```suggestion
					We recommend attending workshops and events (such as HOTH or Hack
					Kitchen), joining a Hack Fam, or asking us questions in the ACM discord!
```

Currently there's a space rendered before the comma.

File: src/components/PhotoPage/FAQSection.js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+				<>
+					Check out the <Link href="https://hack.uclaacm.com/archive">archive</Link> page!
+				</>
+		},
+		{
+			question: `How do I join your team?`,
+			answer:
+				<>
+					We start recruitment for internship positions every fall
+					, following the ACM general meeting (which you should attend)!
```
Comment (TimothyGu) [inline src/components/PhotoPage/FAQSection.js:143]: Same here
```suggestion
					We start recruitment for internship positions every fall,
					following the ACM general meeting (which you should attend)!
```

File: src/components/PhotoPage/FAQSection.js (hunk 3 above)
Comment (jodymlin) [reply src/components/PhotoPage/FAQSection.js:128]: We might not always be holding events Hack Fam, HOTH, or Hack Kitchen every quarter. To keep maintaining this simple we should just saying something like "check out our events page to see what events we have this quarter! Everyone is able to join!" Be sure to emphasize that _anyone_ can join. That's something we always try to let people know.

File: src/components/PhotoPage/FAQSection.js [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+					Every intern becomes an officer following a quarter of training.
```
Comment (jodymlin) [inline src/components/PhotoPage/FAQSection.js:144]:
```suggestion
					After a quarter of shadowing officers, interns may have the opportunity to become full-time officers. All of our current officers started as interns and we highly encourage you to look out for applications in the Fall!
```
Or something similar to that 😃 

For "legal" reasons we shouldn't guarantee that all interns become officers. While historically all interns have become officers, this has never actually been a guarantee.

File: src/components/PhotoPage/FAQSection.js [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+				</>
+		},
+		{
+			question: `I have some other questions.`,
+			answer:
+				<>
+					Send us an email at <Link href='mailto:uclaacmhack@gmail.com'>uclaacmhack@gmail.com!</Link>
```
Comment (jodymlin) [inline src/components/PhotoPage/FAQSection.js:151]:
```suggestion
					Send us an email at <Link href='mailto:uclaacmhack@gmail.com'>uclaacmhack@gmail.com</Link>!
```

**Review by jodymlin — CHANGES_REQUESTED** [review]

**Comment by khxia** [comment]
Let's make the "Team" section on the navigation tab be named "About" instead.

**Comment by khxia** [comment]
This looks good to me. But we decided in the last meeting that we should talk a little more about where to put the FAQ, so I'll wait until then to approve it.

**Review by khxia — APPROVED** [review]

**Pull request merged**